airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/jdbc/CreateDropTableOperation.java

```diff
@@ -0,0 +1,22 @@
+package io.airbyte.integrations.destination.jdbc;
+
+import io.airbyte.db.jdbc.JdbcDatabase;
+import io.airbyte.integrations.destination.NamingConventionTransformer;
+import java.util.UUID;
+
+public class CreateDropTableOperation {
```

Comment:
This class seems to be practically static (there is no state in the class instance) unless I am missing something. What is the purpose of replacing static method `attemptSQLCreateAndDropTableOperations` with class instantiation and a call to instance method?